### PR TITLE
Upkeep

### PR DESCRIPTION
**Pull request by dunkenwg**

### CODE REVIEW

File: R/functions_check.R [hunk 1]
```diff
@@ -31,7 +31,7 @@ chk_population <- function(x, x_name = NULL) {
 
 chk_parameters <- function(tmax, k, Linf, t0, k2, Linf2, L2, Wb, Ls, Sp, es,
                            tR, Rk, BH, fb, n, nL, Ln, Sm, pi, Lv, Vp, Llo, Lup, 
-                           rho, Hm, Nc, Wa, fa, Rmax, q) {
+                           rho, Hm, Nc, Wa, fa, Rmax, q, RPR) {
```
Comment (dunkenwg) [inline R/functions_check.R:34]: Added `RPR` to `chk_parameters()` as I think it might've been forgotten.

File: R/functions_check.R [hunk 2]
```diff
@@ -156,4 +156,8 @@ chk_parameters <- function(tmax, k, Linf, t0, k2, Linf2, L2, Wb, Ls, Sp, es,
   chk_scalar(q)
   chk_not_any_na(q)
   chk_range(q, c(0, 1))
+  chk_s3_class(RPR, "numeric")
+  chk_scalar(RPR)
+  chk_not_any_na(RPR)
+  chk_range(fa, c(1e-04, 100))
```
Comment (dunkenwg) [inline R/functions_check.R:162]: If `RPR` is kept in `chk_parameters()`, its checks need to be reviewed. Especially for the range.

**Review by dunkenwg — COMMENTED** [review]
Can you take a look at these changes to make sure they're right?

**Comment by sebdalgarno** [comment]
thanks @dunkenwg this is a bit silly but there are actually the same changes already in an unmerged PR from 2023 - 
see #18 
The PR was approved but not merged.
I think it makes sense to merge the fix app PR instead as it includes the same changes + some other ones required to get the shiny app working again

**Pull request closed without merging**